ztools/x509/pkix/json.go

```diff
@@ -175,10 +175,10 @@ func (o *OtherName) MarshalJSON() ([]byte, error) {
 	return json.Marshal(oName)
 }
 
-func (o *OtherName) UnmarshalJSON(b []byte) error {
+func (o *OtherName) UnmarshalJSON(b []byte) (err error) {
 	var oName jsonOtherName
 
-	if err := json.Unmarshal(b, &oName); err != nil {
+	if err = json.Unmarshal(b, &oName); err != nil {
 		return err
```

Comment:
when you're naming return variables, I think it's conventional to simply write `return` which makes it more clear that there are named return variables, rather than `return err` which somewhat defeats the point of a named return variable.